File: content/create-symlink/chdir.go
package main
import (
   "os"
   "path/filepath"
)
func main() {
   var s_wd = os.Getwd()
   var s_pub = os.Getenv("PUBLIC")
   var s_old, s_new string
   // round 1
   s_old = filepath.Join(s_wd, "a.go")
   s_new = filepath.Join(s_pub, "Music")
   os.Chdir(s_new)
   os.Symlink(s_old, "a.go")
   // round 2
   s_old = filepath.Join(s_wd, "b.go")
   s_new = filepath.Join(s_pub, "Videos")
   os.Chdir(s_new)
   os.Symlink(s_old, "b.go")
}

File: content/create-symlink/chdir.php
<?php
function f_join(...$a_elem) {
   return implode(DIRECTORY_SEPARATOR, $a_elem);
}
$s_wd = getcwd();
$s_pub = getenv('PUBLIC');
# round 1
$s_old = f_join($s_wd, 'abs.php');
$s_new = f_join($s_pub, 'Music');
chdir($s_new);
symlink($s_old, 'abs.php');
# round 2
$s_old = f_join($s_wd, 'chdir.php');
$s_new = f_join($s_pub, 'Videos');
chdir($s_new);
symlink($s_old, 'chdir.php');
